crashsimilarity/crash_similarity.py

```diff
@@ -70,10 +71,27 @@ def get_stack_trace_for_uuid(uuid):
     return data['proto_signature']
 
 
-def train_model(corpus):
-    if os.path.exists('stack_traces_model.pickle') and \
-       os.path.exists('stack_traces_model.pickle.docvecs.doctag_syn0.npy'):
-        return gensim.models.Doc2Vec.load('stack_traces_model.pickle')
+def delete_old_models(current_date, force_train):
+    """
+    Get list of trained models inside the models directory,
+    delete all models in case of user forces new training,
+    if not, delete all models except of today's model
+    """
+    utils.create_dir('models')
```

Comment:
I think we can use something like [os.path.isdir](https://docs.python.org/dev/library/os.path.html#os.path.isdir), no need to add a new function in utils, right?